buildscripts/build-managed.cmd

```diff
@@ -40,21 +40,24 @@ set "NUGET_PACKAGES=%__PackagesDir%"
 echo Using CLI tools version:
 dir /b "%__DotNetCliPath%\sdk"
 
-"%__DotNetCliPath%\dotnet.exe" msbuild "%__ProjectDir%\build.proj" /nologo /t:Restore /flp:v=normal;LogFile=build-restore.log /p:NuPkgRid=win7-x64 /maxcpucount /p:OSGroup=%__BuildOS% /p:Configuration=%__BuildType% /p:Platform=%__BuildArch% %__ExtraMsBuildParams%
+set __NugetRuntimeId=win7-x64
+if /i "%__BuildArch%" == "x86" (set __NugetRuntimeId=win7-x86)
```

Comment:
Could you please move these lines to buildvars-setup.cmd ?

Reply:
Sure, should the `set "NUGET_PACKAGES=%__PackagesDir%"` be also removed from this file? (few lines above; it's already defined in buildvars)

Reply:
I think `NUGET_PACKAGES` can be deleted from buildscripts completely. The `NuGet.config` at the root of the repo should take care of everything.